crnk-activiti/src/main/java/io/crnk/activiti/mapper/ActivitiResourceMapper.java

```diff
@@ -177,7 +177,7 @@ private void copyDynamicField(Object resource, BeanAttributeInformation attribut
 				if (toResource) {
 					Object value = variables.get(key);
 					if (value != null) {
-						Object mappedValue = mapValue(value, attribute.getImplementationClass());
```

Comment:
Unfortunately things cannot be renamed for the sake of backward compatiblity. The exception is everything within a "internal" package.

Reply:
yes, just public API, thx!